internal/cmd/skupper/system/nonkube/system_uninstall.go

```diff
@@ -0,0 +1,71 @@
+package nonkube
+
+import (
+	"errors"
+	"fmt"
+	"github.com/skupperproject/skupper/internal/cmd/skupper/common"
+	"github.com/skupperproject/skupper/internal/nonkube/bootstrap"
+	"github.com/spf13/cobra"
+)
+
+type CmdSystemUninstall struct {
+	CobraCmd         *cobra.Command
+	Namespace        string
+	SystemUninstall  func() error
+	CheckActiveSites func() (bool, error)
+	Flags            *common.CommandSystemUninstallFlags
+	forceUninstall   bool
+}
+
+func NewCmdSystemUninstall() *CmdSystemUninstall {
+
+	skupperCmd := CmdSystemUninstall{}
+
+	return &skupperCmd
+}
+
+func (cmd *CmdSystemUninstall) NewClient(cobraCommand *cobra.Command, args []string) {
+	cmd.SystemUninstall = bootstrap.Uninstall
+	cmd.CheckActiveSites = bootstrap.CheckActiveSites
+	cmd.Namespace = cobraCommand.Flag("namespace").Value.String()
+}
+
+func (cmd *CmdSystemUninstall) ValidateInput(args []string) error {
+	var validationErrors []error
+
+	if len(args) > 0 {
+		validationErrors = append(validationErrors, fmt.Errorf("this command does not accept arguments"))
+	}
+
+	if cmd.Flags != nil && !cmd.Flags.Force {
```

Comment:
Install only runs for podman platform, but uninstall runs for docker and linux.

Reply:
fixed, now the allowed platforms are only docker and podman